server/controllers/apply/applicationsController.ts

```diff
@@ -177,7 +177,6 @@ export default class ApplicationsController {
         await this.applicationService.submit(req.user.token, application)
         res.render('applications/confirm', { pageHeading: 'Application confirmation', application })
       } catch (err) {
-        err.data = { detail: err.message }
```

Comment:
You've got a footnote in your commit message, not quite sure where it's suppose to point, but will remove it - maybe you can add it to the PR description in case it's useful.